remote-api/src/main/java/org/alfresco/rest/api/impl/TagsImpl.java

```diff
@@ -244,8 +246,11 @@ public Tag getTag(StoreRef storeRef, String tagId)
 
     public CollectionWithPagingInfo<Tag> getTags(String nodeId, Parameters params)
     {
-		NodeRef nodeRef = validateTag(nodeId);
-
+		NodeRef nodeRef = nodes.validateNode(nodeId);
```

Comment:
This looks like an improvement (assuming the methods do what I think). I'd expect `validateTag(nodeId)` to want the nodeId for a tag, rather than a document/folder.

Reply:
Yes, that's true